Find dependencies in MacPorts prefix. MacPorts will replace the
placeholder with the actual MacPorts prefix.
--- skia/third_party/freetype2/BUILD.gn.orig	2022-05-24 18:06:11.000000000 -0500
+++ skia/third_party/freetype2/BUILD.gn	2022-06-03 12:54:31.000000000 -0500
@@ -15,7 +15,7 @@
 
 if (skia_use_system_freetype2) {
   system("freetype2") {
-    include_dirs = [ "/usr/include/freetype2" ]
+    include_dirs = [ "@PREFIX@/include/freetype2" ]
     libs = [ "freetype" ]
   }
 } else {
--- skia/third_party/harfbuzz/BUILD.gn.orig	2021-11-16 18:33:27.000000000 -0600
+++ skia/third_party/harfbuzz/BUILD.gn	2022-01-30 23:21:32.000000000 -0600
@@ -12,7 +12,7 @@
 
 if (skia_use_system_harfbuzz) {
   system("harfbuzz") {
-    include_dirs = [ "/usr/include/harfbuzz" ]
+    include_dirs = [ "@PREFIX@/include/harfbuzz" ]
     libs = [ "harfbuzz" ]
     if (skia_pdf_subset_harfbuzz) {
       libs += [ "harfbuzz-subset" ]
